server/Dockerfile

```diff
@@ -4,14 +4,26 @@ FROM jboss/base-jdk:8
 # Set the INFINISPAN_SERVER_HOME env variable
 ENV INFINISPAN_SERVER_HOME /opt/jboss/infinispan-server
 
+ENV INFINISPAN_SHA a55ca9cddd5b3373029b780f6530ccdffdc8396f
```

Comment:
Maybe we could extract it from Nexus? https://repository.jboss.org/nexus/content/repositories/public/org/infinispan/infinispan-embedded/8.2.3.Final/infinispan-embedded-8.2.3.Final.jar.sha1